lib/wallaby/experimental/selenium/webdriver_client.ex

```diff
@@ -316,4 +364,15 @@ defmodule Wallaby.Experimental.Selenium.WebdriverClient do
       driver: Wallaby.Experimental.Selenium,
     }
   end
+
+  @doc """
+  Retrieves the text from an alert, prompt or confirm.
+  """
+  @spec alert_text(Session.t) :: {:ok, String.t}
+  defp alert_text(session) do
```

Comment:
I'll comment the doc and leave it there in case the method is becoming public, somehow in the future?! 

Re:
> Another improvement I would like to make (unrelated to this PR though) is to have the driver modules return {:ok, any()} | {:error, any()} instead of just returning the message.

I too prefer the `{:ok|:error, .. `, but in this particular round, I was trying to stay as close as possible to the phantom driver implementation, as I am still new to this code - I know you previous comment is not strictly related to this specific PR, but I'd thought I'd state my opinion too, in support of yours :)